### ISSUE TITLE
Repeat Last Snooze Shortcut

### ISSUE THREAD

**Issue by kabglox** [rohanb10/snoozz-tab-snoozing#99]
If possible, could you please add the ability to, via shortcut, repeat the last snooze? For example, if I snooze a tab for next Wednesday at 2pm, then want to repeat that time for multiple other tabs, I can just type in a shortcut. Saves a lot of time, especially when using custom snooze times. If this worked after setting "repeat" snoozes, too (tabs snoozed daily, weekly, custom, intervals, etc) that would be awesome. This shortcut saves so much time when snoozing multiple tabs for the same date or the same repeated interval. Thanks in advance!

(chrome  99.0.4844.51)

**Comment by rohanb10** [comment]
Yeah I'll add a quick way to repeat the last custom snooze time with a keyboard shortcut.

**Comment by kabglox** [comment]
Amazing, thank you!

**Comment by kabglox** [comment]
This may be a shot in the dark but any chance this feature could still be
added?
